### DISCUSSION TITLE
Error editing Jellyfin button

### DISCUSSION THREAD

**Discussion by shishio-git** [linuxserver/Heimdall#751] [App issues]
Hi,
I run Heimdall 2.2.2 on docker, all good. But when I edit the Jellyfin button it happens in the following error:

```
ErrorException (E_ERROR)
Undefined property: stdClass::$availablestats (View: /config/www/SupportedApps/Jellyfin/config.blade.php) (View: /config/www/SupportedApps/Jellyfin/config.blade.php) (View: /config/www/SupportedApps/Jellyfin/config.blade.php)
```
Thx.

**Comment by dontoverthinkit** [comment]
@shishio-git I was getting something similar on a couple of apps (ombi, and - I *think* the other one was jellyfin) - I hadn't played with heimdall in a while so assumed I had something cached after updates, so I deleted the entry for each, and re-added (to heimdall) and had no further issues. I don't have much more detail sorry as I wasn't really paying too much attention assuming the issue was with the state of my setup.

**Comment by hughesjs** [comment]
Same issue:

> Undefined property: stdClass::$availablestats (View: /config/www/SupportedApps/Jellyfin/config.blade.php) (View: /config/www/SupportedApps/Jellyfin/config.blade.php) (View: /config/www/SupportedApps/Jellyfin/config.blade.php)

**Comment by drissical** [comment]
Getting the same error as well with Jellyfin.
```
Undefined property: stdClass::$availablestats 
(View: /config/www/SupportedApps/Jellyfin/config.blade.php) 
```

**Comment by racingpassion** [comment]
This problem is still persisting for me, has anyone got any ideas.

**Comment by Kruemmelspalter** [comment]
I also have that problem (running both in docker)

**Comment by elvismercado** [comment]
I am having the same issue

**Comment by shizunge** [comment]
I got the same issue.

**Comment by ChristophorusReyhan** [comment]
Same issue
tracing $this->config only results in:
```
stdClass Object
(
    [enabled] => 1
    [override_url] => http://10.0.0.107:8096
    [password] => strongpasswordobviously
    [url] => http://10.0.0.107:8096
)
```
no availablestats in sight. I can't trace where config is set.

**Reply by ChristophorusReyhan** [reply]
Oh found it, it's set from `SupportedApps/Jellyfin/config.blade.php`. 

In item edit, you have to select the items in "Stats to show". Then it will show them.

Works fine now.

**Reply by ChristophorusReyhan** [reply]
However, it should preselect something or display things like "No stats selected" in the homepage if the user doesn't select anything. Not just blank page and user wondering why it doesn't show anything.
